packages/ui/src/molecules/Dropdown/DropdownItem.tsx

```diff
@@ -0,0 +1,67 @@
+import type { ButtonHTMLAttributes } from 'react'
+import React, { useImperativeHandle, forwardRef, useRef, useState } from 'react'
+
+import { useDropdown } from './hooks/useDropdown'
+
+export interface DropdownItemProps<T = HTMLButtonElement>
+  extends ButtonHTMLAttributes<T> {
+  /**
+   * ID to find this component in testing tools (e.g.: cypress, testing library, and jest).
+   */
+  testId?: string
+}
+
+const DropdownItem = forwardRef<HTMLButtonElement, DropdownItemProps>(
+  function Button(
+    { children, onClick, testId = 'store-dropdown-item', ...otherProps },
+    ref
+  ) {
+    const { dropdownItemsRef, selectedDropdownItemRef, close } = useDropdown()
+    const [dropdownItemIndex, setDropdownItemIndex] = useState(0)
+    const dropdownItemRef = useRef<HTMLButtonElement>()
+
+    const addToRefs = (el: HTMLButtonElement) => {
+      if (el && !dropdownItemsRef?.current.includes(el)) {
+        dropdownItemsRef?.current.push(el)
+        setDropdownItemIndex(
+          dropdownItemsRef?.current.findIndex((element) => element === el) ?? 0
+        )
+      }
+
+      dropdownItemRef.current = el
+    }
+
+    const onFocusItem = () => {
+      selectedDropdownItemRef!.current = dropdownItemIndex
+      dropdownItemsRef?.current[selectedDropdownItemRef!.current]?.focus()
+    }
+
+    const handlerOnClickItem = (
```

Comment:
I think we can follow the pattern when naming the handlers. I think `handleOnClickItem` sounds better. It could also be `onClickItemHandler,` but I'll suggest `handleOnClickItem` to keep consistent. What do you think?

```suggestion
    const handleOnClickItem = (
```

Reply:
I agree! 😄